.github/workflows/rust.yml

```diff
@@ -19,11 +19,13 @@ jobs:
         components: rustfmt, clippy
         override: true
     - name: Install deps
-      run: sudo apt-get install -y clang-12 libelf-dev
+      run: sudo apt-get install -y clang-12 libelf-dev zlib1g-dev
```

Comment:
Perhaps this worked before because clang or libelf require it, too. Let's make it explicit

Reply:
Does the capable example with static libelf and libz fail to compile without zlib1g-dev explicitly added?

Reply:
It should still work as it's already installed on the machine. It includes [both the dynamic library and the static archive](https://packages.ubuntu.com/focal/amd64/zlib1g-dev/filelist). Added it to make the dependencies clearer for people reading the CI setup